pkg/antctl/raw/multicluster/deploy/deploy_helper.go

```diff
@@ -0,0 +1,143 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package deploy
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+
+	"github.com/spf13/cobra"
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer/yaml"
+	yamlutil "k8s.io/apimachinery/pkg/util/yaml"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/restmapper"
+
+	"antrea.io/antrea/pkg/antctl/raw"
+)
+
+const (
+	downloadURL          = "https://github.com/antrea-io/antrea/releases/download"
+	leaderGlobalYAML     = "antrea-multicluster-leader-global.yml"
+	leaderNamespacedYAML = "antrea-multicluster-leader-namespaced.yml"
+	memberYAML           = "antrea-multicluster-member.yml"
+)
+
+func generateManifests(role string, version string) []string {
+	var manifests []string
+	if role == "leader" {
+		manifests = []string{
+			fmt.Sprintf("%s/%s/%s", downloadURL, version, leaderGlobalYAML),
+			fmt.Sprintf("%s/%s/%s", downloadURL, version, leaderNamespacedYAML),
+		}
+	} else {
+		manifests = []string{
+			fmt.Sprintf("%s/%s/%s", downloadURL, version, memberYAML),
+		}
+	}
+
+	return manifests
+}
+
+func deploy(cmd *cobra.Command, role string, version string, namespace string) error {
+	kubeconfig, err := raw.ResolveKubeconfig(cmd)
+	if err != nil {
+		return err
+	}
+	restconfigTmpl := rest.CopyConfig(kubeconfig)
+	raw.SetupKubeconfig(restconfigTmpl)
+
+	k8sClient, err := kubernetes.NewForConfig(kubeconfig)
+	if err != nil {
+		return err
+	}
+	dynamicClient, err := dynamic.NewForConfig(kubeconfig)
+	if err != nil {
+		return err
+	}
+	manifests := generateManifests(role, version)
+
+	for _, manifest := range manifests {
+		resp, err := http.Get(manifest)
+		if err != nil {
+			return err
+		}
+		b, err := ioutil.ReadAll(resp.Body)
+		if err != nil {
+			return err
+		}
+
+		content := string(b)
+		if role == "leader" && strings.Contains(manifest, "namespaced") {
+			content = strings.ReplaceAll(content, "changeme", namespace)
+		}
```

Comment:
How do you support namespace change for member cluster? I didn't see any related codes.
The namespace is kube-system by default, we may just use the default one if user doesn't use the namespace option.

Reply:
Sorry I forgot to change the member cluster namespace,  the latest code changes it now.